### PR TITLE
Matt Leach Project Descriptions

### PR DESCRIPTION
**Pull request by MILeach**
Project descriptions for "The Hole in the Road" and "Flood Modelling - Festival of the Mind"

### CODE REVIEW

File: _data/projects.csv [hunk 1]
```diff
@@ -29,10 +29,10 @@ id,key,title,long_title,tech_methods,rses,start,end,department,level,show
 53,ramp,RAMP SCRC Covid Support,RAMP SCRC Covid Support,"python, agile, software testing, continuous integration, documentation",Robert Turner,01/05/2020,01/08/2020,Computer Science,2,1
 55,,IFPRI,IFPRI,,David Wilby,01/07/2020,01/08/2020,Computer Science,2,1
 56,,Opening Up Minds,Opening Up Minds,,Anna Krystalli,18/01/2021,18/06/2021,Psychology,2,1
-57,,Flood Modelling Festival of the Mind,Flood Modelling Festival of the Mind,,"Matt Leach, Paul Richmond",01/07/2020,01/09/2020,Civil Engineering,2,1
+57,floodfotm,Flood Modelling Festival of the Mind,Flood Modelling Festival of the Mind,"GPU, unreal engine, FLAMEGPU","Matt Leach, Paul Richmond",01/07/2020,01/09/2020,Civil Engineering,2,1
```
Comment (willfurnass) [inline _data/projects.csv:32]: Decrement 2 to 1

File: _data/projects.csv [hunk 2]
```diff
@@ -29,10 +29,10 @@ id,key,title,long_title,tech_methods,rses,start,end,department,level,show
...
 58,ratesetter,RateSetter / Merseyrail,RateSetter / Merseyrail,"C++, FLAME GPU, ABM",Twin Karmakharm,18/11/2019,01/12/2020,Mechanical Engineering,2,1
 64,scrc-pipeline,SCRC Follow on,Covid epidemiology data pipeline,"python, agile, software testing, continuous integration, documentation",Robert Turner,01/01/2021,01/01/2022,External,1,1
-65,,Hole in the Road,Hole in the Road,,Matt Leach,24/09/2020,13/10/2020,Computer Science,2,1
+65,holeintheroad,Hole in the Road,Hole in the Road,"VR, unity",Matt Leach,24/09/2020,13/10/2020,Computer Science,2,1
```
Comment (willfurnass) [inline _data/projects.csv]: Decrement 2 to 1

**Review by willfurnass — APPROVED** [review]
Nice descriptions.  See minor comments but LGTM.

**Review by Robadob — COMMENTED** [review]
As these are both graphical projects, could you include photos in the descriptions?

Don't think any projects have them yet, but the same was suggested of my unfinished Primage work by @willfurnass .

**Comment by willfurnass** [comment]
Sorry, you've also got merge conflicts to resolve re [_data/projects.csv](_data/projects.csv)

File: _data/projects.csv (hunk 2 above)
Comment (willfurnass) [reply _data/projects.csv]: Done this as part of rebase

**Comment by willfurnass** [comment]
> Sorry, you've also got merge conflicts to resolve re [_data/projects.csv](_data/projects.csv)

I've fixed this for you.

**Comment by willfurnass** [comment]
@MILeach If you think it'll be tricky finding accompanying graphics or think just linking to pages with nice graphics will be fine then it would be good to get this PR merged before the event tomorrow.

**Comment by Robadob** [comment]
http://www.storyingsheffield.com/stories/the-virtual-hole-in-the-road/

Is the image from this article one you produced? Would probably be appropriate.

File: _data/projects.csv (hunk 2 above)
Comment (MILeach) [reply _data/projects.csv]: Thanks Will

**Comment by MILeach** [comment]
> http://www.storyingsheffield.com/stories/the-virtual-hole-in-the-road/
> 
> Is the image from this article one you produced? Would probably be appropriate.

I think this particular image would have been produced by Human, we were working with them but I'm unsure what the license would be. I'll see if I can find one that I produced.

**Comment by Robadob** [comment]
> > http://www.storyingsheffield.com/stories/the-virtual-hole-in-the-road/
> > Is the image from this article one you produced? Would probably be appropriate.
> 
> I think this particular image would have been produced by Human, we were working with them but I'm unsure what the license would be. I'll see if I can find one that I produced.

Unsure if @willfurnass would be happy with a video. There's a proper promo video on dept youtube page.

https://www.youtube.com/watch?v=gBjP4KeLZU4

**Comment by willfurnass** [comment]
@MILeach Happy to merge this?

**Comment by MILeach** [comment]
> @MILeach Happy to merge this?

Yes of course, sorry I had completely forgotten about it!

**Pull request merged**